chsdi/templates/htmlpopup/oerebkataster.mako

```diff
@@ -15,10 +15,10 @@ def sanitise_url(url):
 def get_xml(path):
     list_egrid = []
     try:
-      response = requests.get(path)
+      response = requests.get(path, verify=False) # verify=False to get it working for grison
```

Comment:
I would not set it to False this is a security issue. The issue is due to the Root Certificate used by *.geo.gr.ch (SwissSign AG), it seems that not all browsers know it (at least Firefox 79.0 on ubuntu don't but Chrome 84.0.4147.125 on ubuntu does).
I would suggest to get the Root Certificate from SwissSign and install it on the machine running the mf-chsdi3. See https://requests.readthedocs.io/en/master/user/advanced/#ssl-cert-verification

Reply:
This sounds like a professional solution